tldw_Server_API/tests/sandbox/test_artifact_traversal_integration.py

```
@@ -0,0 +1,73 @@
+import time
+from typing import Any
+
+import pytest
+
+
+@pytest.mark.integration
+def test_artifact_traversal_rejected_under_uvicorn(tmp_path) -> None:
+    # Only run if uvicorn is available
+    try:
+        import uvicorn  # type: ignore
+    except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)